schema/configuration-structure.asn

```diff
@@ -32,27 +34,110 @@ BEGIN
          customOctetString SEQUENCE OF CustomOctetString,
          ...
       }
+      ...
```

Comment:
```suggestion
      },
      ...
```

It's probably worth your time to get `make test` to work on your system so you don't end up pushing files with syntax errors.